addons/event_crm_sale/views/event_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_event_registration_ticket_form_inherited" model="ir.ui.view">
```

Comment:
event_registration_ticket_view_form
and align original view. id should be the same